torchvision/csrc/PSROIPool.h

```diff
@@ -81,13 +81,14 @@ class PSROIPoolFunction : public torch::autograd::Function<PSROIPoolFunction> {
     ctx->saved_data["pooled_height"] = pooled_height;
     ctx->saved_data["pooled_width"] = pooled_width;
     ctx->saved_data["input_shape"] = input.sizes();
-    auto result = PSROIPool_forward(
-        input, rois, spatial_scale, pooled_height, pooled_width);
+    at::AutoNonVariableTypeMode g;
+    auto result =
+        ps_roi_pool(input, rois, spatial_scale, pooled_height, pooled_width);
     auto output = std::get<0>(result);
-    auto channel_mapping = std::get<1>(result);
-    ctx->save_for_backward({rois, channel_mapping});
-    ctx->mark_non_differentiable({channel_mapping});
-    return {output, channel_mapping};
+    auto mapping_channel = std::get<1>(result);
```

Comment:
In case anybody wonders about this change, in this codebase we used both mapping_channel and channel_mapping. Here I picked mapping_channel, but later on #2926 I replace it everywhere with `channel_mapping` which I think makes more sense.